### ISSUE TITLE
Flip stream

### ISSUE THREAD

**Issue by scns** [jneilliii/OctoPrint-YouTubeLive#6]
In OctoPrint it's possible to flip the webcam feed.

![schermafbeelding 2018-01-29 om 21 04 45](https://user-images.githubusercontent.com/7148071/35531726-292681f8-0538-11e8-8ff4-e299127eb1e3.png)

These settings are not used in the Youtube Feed. How can I transfer these settings into the feed?

**Comment by jneilliii** [comment]
Yeah, I figured this request would be on its way when I saw the streams posted to my account being upside down. It may take me some time to figure that out as I'm not super familiar with ffmjpeg, but essentially you need to modify the entry.sh file that is cloned from github for the proper parameters to the ffmjpeg command.

**Comment by scns** [comment]
Nice. Thanks for the great work!

Works fine now!

Met vriendelijke groet,


*Maarten Schmeitz*






2018-01-31 2:16 GMT+01:00 jneilliii <notifications@github.com>:
